lib/csv.ex

```diff
@@ -236,6 +248,14 @@ defmodule CSV do
        escape_max_lines: escape_max_lines
      ).message}
   end
+  defp yield_or_inline!({:error, EscapeSequenceError, escape_sequence, index, raw_line}, escape_max_lines) do
+    {:error,
+     EscapeSequenceError.exception(
+       escape_sequence: escape_sequence,
+       line: index + 1,
+       escape_max_lines: escape_max_lines
+     ).message, raw_line}
```

Comment:
This is breaking backwards compatibility if I interpret it correctly. If someone has existing code that depends on the length of the tuple being returned, they will get a compilation error. We should either try to avoid it by adding the raw line to the message, or reserve this for a major version upgrade.

Reply:
I included the `:raw_line_on_error` option (defauls to false) so that only when set to true the extra `raw_line` will return on the error tuple,  so I'd say it won't break backwards compatibility.